api/pom.xml

```diff
@@ -36,6 +36,12 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Pulled in the persist module here because I want to use JOOQ classes in a factory method for one of the DTOs, if we don't want to do this for whatever reason we can just make a factory class in service